debug/debug.go

```diff
@@ -27,7 +27,7 @@ type ProfileName string
 
 const (
 	pair = 2
-	// PPROFDumpTimeout when dumping PPROF data, set an upper bound on the time it can take to log.
+	// PPROFDumpTimeout when dumping PPROF data, set an upper bound on the time it can take to log. (default 15s).
```

Comment:
nit: the newly added text seems to be excessive, considering that we can immediately the value of the const.